apps/api/test/setup/index.js

```diff
@@ -66,7 +66,7 @@ const createViaDelegation = (api, provider) => async (keyUri, baseNonce) => {
     tx.signAndSend(provider, { nonce: baseNonce }, sendStatusCb(resolve)),
   );
 
-  const msaId = await api.query.msa.publicKeyToMsaId(delegator.address);
+  const msaId = await api.query.msa.publicKeyToMsaId(delegator);
```

Comment:
Seems like this might be a bug?
```suggestion
  const msaId = await api.query.msa.publicKeyToMsaId(delegator.address);
```

Also is the code for Alice claiming a handle missing from here?

Reply:
That fixed the error! Don't know why that change happened.

Yes! Alice creates a handle starting on line 152.